woocommerce/woocommerce
# Fix the welcome modal and remove the calypso-specific one

### Description

This PR aims to
- Fix an issue with showing the "Welcome" modal, and;
- Remove the welcome modal that it's associated with Calypso. 

Currently, to determine if the modal is dismissed, the check is:
```js
const welcomeModalDismissed =
			getOption( WELCOME_MODAL_DISMISSED_OPTION_NAME ) !== 'no';
```
This is `true` if the option value **is empty**, which _is_ the case on new DB installations. Thus, the system always considers the modal as _dismissed_ and _never_ shows it.

We could write a more defensive check, like this one:
```js
const welcomeModalDismissed =
			getOption( WELCOME_MODAL_DISMISSED_OPTION_NAME ) === 'yes';
```

### How to test the changes in this Pull Request:

1. Create a new WooCommerce site
2. Open WooCommerce > Home for the first time
3. See the welcome modal.

#### Testing on existing installations
1. Ensure that there is no option named `woocommerce_task_list_welcome_modal_dismissed` in your `wp_options` table. If there is one, just delete it.
2. Open WooCommerce > Home
3. See the welcome modal.

![Welcome modal](https://cdn-std.droplr.net/files/acc_1208648/RkVVqv)
### Other information:




### FOR PR REVIEWER ONLY:
